firedantic/_async/model.py

```diff
@@ -53,6 +69,14 @@ def _get_col_ref(cls, name: Optional[str]) -> AsyncCollectionReference:
     return collection
 
 
+def collection_index(*fields: IndexField) -> IndexDef:
```

Comment:
Just thinking that would it make sense to have some docstring for the `collection_index` and `collection_group_index` with examples to use them or not. On the other hand they're explained in the README. But wouldn't necessarily be bad to have some description and examples of them. Feel free to resolve as-is or do some changes.